docs/modules/clients/pages/java.adoc

```diff
@@ -16,10 +17,10 @@ NOTE: Where there are specific differences between {java-client} and {java-clien
 
 // check production recommendation
 
-Both native ({java-client}) and standalone ({java-client-new}) clients enable you to use the Hazelcast API, with this page explaining any differences or technical details that affect usage. This page should be read alongside the respective Javadoc-generated API documentation available from within your IDE and the following links:
+Both platform ({java-client}) and standalone ({java-client-new}) clients enable you to use the Hazelcast API, with this page explaining any differences or technical details that affect usage. This page should be read alongside the respective Javadoc-generated API documentation available from within your IDE and the following links:
 
 * https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc[Hazelcast {java-client} API documentation]
-* https://docs.hazelcast.org/hazelcast-java-client/{page-latest-supported-java-client-new}/javadoc[Hazelcast {java-client-new} API documentation]
+* https://docs.hazelcast.org/hazelcast-java-client-docs/{page-latest-supported-java-client-new}/apidocs[Hazelcast {java-client-new} API documentation]
```

Comment:
Link works ok but not sure if we still have control over the generated url as I think the '-docs' is redundant and we're using /javadoc in 1st case and /apidocs in second. Ideally we should be consistent but no major objection if this is what we are sticking with.

Reply:
OK, changed as suggested the folder names at the docs server.

Reply:
Thanks, I think that's better